RELEASE_NOTES.md

```diff
@@ -299,6 +299,38 @@ Our underlying SSH implementation has been updated to support the new ed25519 SS
 
 Chef Solo's `--delete-entire-chef-repo` option has been extended to work in Local Mode as well. Be warned that this flag does exactly what it states, and when used incorrectly, can result in loss of work.
 
+### sysctl now accepts a comments parameter
+
+The sysctl resource has been updated to allow additional (optional) comments. Comments can be passed as an array or a string. Any comments provided are prefixed with '#' signs and precede the sysctl setting in generated files.
+
+An example:
+
+```
+sysctl 'vm.swappiness' do
+  value 10
+  comment [
+     "define how aggressively the kernel will swap memory pages.",
+     "Higher values will increase aggressiveness",
+     "lower values decrease the amount of swap.",
+     "A value of 0 instructs the kernel not to initiate swap",
+     "until the amount of free and file-backed pages is less",
+     "than the high water mark in a zone.",
+     "The default value is 60."
+    ]
+end
+```
+
+which results in `/etc/sysctl.d/99-chef-vw.swappiness.conf` as follows
+```
+# define how aggressively the kernel will swap memory pages.
+# Higher values will increase aggressiveness
+# lower values decrease the amount of swap.
+# A value of 0 instructs the kernel not to initiate swap
+# until the amount of free and file-backed pages is less
+# than the high water mark in a zone.
+# The default value is 60.
+vm.swappiness = 10
+
```

Comment:
You appear to be missing ending codeblock ticks.

Reply:
ah, you are of course correct. Fixed, will push shortly.